pkg/registry/core/service/storage/storage_test.go

```diff
@@ -4787,10 +4797,24 @@ func TestCreateReallocation(t *testing.T) {
 			svc2.Spec.ClusterIPs = createdSvc.Spec.ClusterIPs
 			svc2.Spec.Ports = createdSvc.Spec.Ports
 
+			// Create again
```

Comment:
Commit `De-layer Service REST: Delete`
I know it might be an overkill. But i would like to add change service name in the second create. It is more likely the scenario that people will use and most importantly i would like to confuse the allocator/repair loop.

Reply:
ACK